site/docs/components/_content-template/index.mdx

```diff
@@ -0,0 +1,234 @@
+
+<!--
+  Bulleted list of scenarios where this component is not appropriate. Where
+  possible, include a link to an alternative component that should be used in
+  it's place
+-->
+
+- ...
+- ...
+
+### Content
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  Add any component-specific content guidance here, use visual examples where
+  appropriate.
+
+  For example, a Button component might include guidance on recommended terms to
+  use for the label text, recommended maximum character counts, etc.
+-->
+
+### Accessibility
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  Describe any considerations the users may need to take into account when using
+  the component. For example:
+
+  - Accessibility decisions that deviate from industry standard. (e.g., we've
+    introduced an arrow key navigation, where usually is only tab).
+  - Accessibility requirements that drive design (e.g., how multi-column forms
+    should wrap).
+  - Best practices regarding accessible content (e.g., error message
+    announcements must not rely on descriptions that require the user to
+    visually or audibly perceive content).
+  - Best practices regarding implementation (e.g., to make sure this component
+    is accessible be sure to provide a relevant alt text).
+-->
+
+### References
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  If there are industry standards for this component, list them in this
+  section. Include links out to useful articles.
+-->
+
+- [Relevant industry standard](https://example-standard.org/)
+- [Useful article](https://some-site.com/article/bla)
+- ...
+
+## Configurations
+
+<!--
+  A visual list of possible configurations of the component.
+
+  The purpose is to convey the main features and/or usage options of the
+  component (e.g. the Data Grid's striped rows might be a "configuration" to
+  showcase here). The configurations listed here should aim to be comprehensive,
+  but not necessarily exhaustive (some complex components may have vast numbers
+  of possible permutations, so listing out each one would be impossible and
+  overwhelming to readers).
+
+  Each confguration should have its own section following the structure outlined
+  below:
+-->
+
+### Configuration title (e.g. Striped rows)
+
+<!--
+  A brief description of what this configuration provides to users and when it
+  should be used.
+-->
+
+<!--
+  A live example of the component configuration, with access to the correspoding
+  source code.
+
+  In Storybook this is done using something like:
+
+  <Canvas>
+    <Story id="..." />
+  </Canvas>
+-->
+
+## Designing
+
+### Best practices
+
+<!--
+  Add a bulleted list of best practices and princples that are relevant to this
+  component. Where other pages on the site explain those principles in more
+  detail, consider linking to them rather than repeating the explanation here.
+-->
+
+- ...
+- ...
+
+### Anatomy
```

Comment:
@shey-v & @sion-docs Should the Anatomy be moved up into the "Overview" section perhaps? I think it could be useful for content folks too, since it could also visualise areas where text is displayed. So, when the "Content" section says something like "limit Button labels to 99 characters", they could refer to the anatomy diagram to see exactly which part of the component we're referring to.

Reply:
I like the idea of putting it there, are we dispensing with Designing and Developing as mentioned previously?

Reply:
Hmm.. maybe there needs to be a bigger re-jig.

How about this:

* **Overview** (provide a high-level overview of the component's purpose and structure
    * Live example
    * Brief description
    * **Anatomy**
* **Usage** (more detail to help people understand when and how to use the component)
    * **Dos**
    * **Dont's**
    * **Best practices** (can this be merged with or replace "References" below?)
    * **Content**
    * **Accessibility**
    * **References**
* **Configurations** (collection of live examples showcasing key features or config options of the component)
    * **_[Config example 1]_**
    * **_[Config example 2]_**
    * ...

* **Figma** (info and resources specific to the Figma version of this component)
    * Link to stickersheet
* **React** (props table and resources specific to the React version of this component)
    * **Import**
    * **Props**

Reply:
I'm happy to merge Best practices and References under Best practices, I'm not sure how many components will have content for References.

Reply:
I think that's a tough call to make without seeing the mock-ups. If the overview section is the page header, above tabs, i wouldn't put the anatomy in there. It's not high enough priority. 

What's the difference between Dos/Don'ts and Best practices? Aren't they the same thing? Suggest we merge those, rather than with References.

Can we lead with configurations? Feels more compelling if the examples are the first thing that you see on the page, rather than blocks of usage text. 

So i'd throw in a proposal:

- Overview (eg and description)
- Configs
- Usage
  - Dos/Dont's
  - Content
  - Accessibility
  - References
- Styling
  - Anatomy (serves as a key to the below styling attributes)
  - Spacing/Size (TBD)
  - Tokens (TBD)
  - +other design content shey had outlined.
- Code
  - Importing
  - Props props props

Reply:
Your proposal looks nice, @bhoppers2008. I'm happy to go with that for now.

My only slight question mark is: If design resources like a Figma stickersheet live under a section/tab/whatever called "Styling", is that where someone would go looking for it?

However, we can speculate here until the cows come home :-) So, let's go with this for now. Once we begin filling in this template and adjusting our page designs accordingly, we'll be able to begin testing it with users.

It may well be fine as is. And if it's not, I suspect it's just a case of moving a section somewhere else or renaming a title. So, worst case scenario we need to do some find-and-replace on a bunch of `.mdx` files at some point. I can live with that :-)

Reply:
Totally agree.

A designer made a comment the other day that we don't do a good job of supporting their working practices... so maybe things like the figma link, repo link, site link, could live somewhere in the header, so they're front and centre and don't need any navigation. Like you say... something to consider.

Reply:
Just pushed `Outline v97.final.FINAL-for-real-this-time.pdf` :-)

If you're all happy please "approve" this PR. Thanks!